Serialization.NET/Segment/Serialization/JsonUtility.cs

```diff
@@ -83,8 +83,8 @@ private static char ToHexChar(int i)
             }
         }
 
-        public static string ToJson(object value, bool pretty = false) => JsonConvert.SerializeObject(value, pretty ? Formatting.Indented : Formatting.None);
+        public static string ToJson(object value) => JsonSerializer.Serialize(value);
 
-        public static T FromJson<T>(string json) => JsonConvert.DeserializeObject<T>(json, new JsonSerializerSettings {DateParseHandling = DateParseHandling.None});
+        public static T FromJson<T>(string json) => JsonSerializer.Deserialize<T>(json);
```

Comment:
we need to find the corresponding `DateParseHandling` option in system.text.json. this will impact the date format we sent over to the server, causing issues on server side analysis.